src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs

```diff
@@ -132,6 +132,19 @@ private void ConvertCore(string basePath)
                             AbsolutePathInTocPageFileTransformer(tocPageFilePath);
                             htmlModels.Insert(0, new HtmlModel { Title = _pdfOptions.TocTitle, HtmlFilePath = tocPageFilePath });
                         }
+
+                        var coverPages = FindCoverPagesInManifest(manifest);
+                        foreach (ManifestItem coverPage in coverPages)
+                        {
+                            var coverPageFilePath = Path.Combine(basePath, coverPage.SourceRelativePath);
+                            var coverPageHtmlFileName = $"{Path.GetFileNameWithoutExtension(coverPageFilePath)}.html";
```

Comment:
DocFX can use [src and dest in file mapping](https://dotnet.github.io/docfx/tutorial/docfx.exe_user_manual.html#41-array-format) to make output path different with input path. So the accurate logic to get output HTML path is like:

    var coverPageHtmlFilePath = coverPage.OutputFiles["html"].RelativePath

BTW, we shall first determine if we need to add cover page into manifest.